NitroxModel-Subnautica/DataStructures/Surrogates/ColorSurrogate.cs

```diff
@@ -1,26 +0,0 @@
-﻿using System.Runtime.Serialization;
-using NitroxModel.DataStructures.Surrogates;
-using UnityEngine;
-
-namespace NitroxModel_Subnautica.DataStructures.Surrogates
-{
-    public class ColorSurrogate : SerializationSurrogate<Color>
-    {
```

Comment:
If it's unused, is there a reason to have kept QuaternionSurrogate & Vector3Surrogate ?

Reply:
Still used in BatchCellsParser